infoschema/tables.go

```diff
@@ -835,6 +835,7 @@ var tableProcesslistCols = []columnInfo{
 	{name: "DISK", tp: mysql.TypeLonglong, size: 21, flag: mysql.UnsignedFlag},
 	{name: "TxnStart", tp: mysql.TypeVarchar, size: 64, flag: mysql.NotNullFlag, deflt: ""},
 	{name: "RESOURCE_GROUP", tp: mysql.TypeVarchar, size: resourcegroup.MaxGroupNameLength, flag: mysql.NotNullFlag, deflt: ""},
+	{name: "SESSION_ALIAS", tp: mysql.TypeString, size: 64, flag: mysql.NotNullFlag, deflt: ""},
```

Comment:
`mysql.TypeString` will be generated as `binary`, please change it to `TypeVarchar`.